kubetest2-tf/data/vpc/node/provider.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "~> 1.50.0"
+      version = ">= 1.12.0"
```

Comment:
why the version change? Is this tested with latest tf?

Reply:
Yes I have kept this version the same as that used by powervs